### PR TITLE
[DNM] Unrc aks operator v1.0.8

### PR DESCRIPTION
**Pull request by a-blender**
## Issue: 

https://github.com/rancher/rancher/issues/40787

## Problem

Unrc aks operator from `v1.0.8-rc2` to `v1.0.8` for Rancher 2.6.11 release. @pgonin the release team just wanted to get this moved along tonight ASAP.

### CODE REVIEW

File: charts/rancher-aks-operator-crd/100.0.7+up1.0.8/templates/crds.yaml [hunk 1]
```diff
@@ -122,6 +122,9 @@ spec:
                     vmSize:
                       nullable: true
                       type: string
+                    vnetSubnetID:
+                      nullable: true
+                      type: string
```
Comment (stormqueen1990) [inline charts/rancher-aks-operator-crd/100.0.7+up1.0.8/templates/crds.yaml:127]: The `vnetSubnetID` block seems to be absent from the original RC'd chart: https://github.com/rancher/charts/blob/48ee4d031658e4d2963ad9e2dc7787dcd63aba30/charts/rancher-aks-operator-crd/100.0.7%2Bup1.0.8-rc2/templates/crds.yaml#L119-L130

Is this a change we want in the released chart?

**Review by stormqueen1990 — COMMENTED** [review]
Mostly LGTM but I have one question about the `vnetSubnetID` block in the `crds.yaml` file.

File: charts/rancher-aks-operator-crd/100.0.7+up1.0.8/templates/crds.yaml (hunk 1 above)
Comment (a-blender) [reply charts/rancher-aks-operator-crd/100.0.7+up1.0.8/templates/crds.yaml:127]: I believe so. If you compare that rc with the latest release/v1.0 https://github.com/rancher/aks-operator/compare/v1.1.0-rc2...release/v1.0, you'll see the highlander team put in this PR https://github.com/rancher/aks-operator/pull/80 which adds the `vmSubnetId` to the node pool in the aks operator. The PR description specifies that those updates/fixes are needed in rancher 2.6.

Comment (stormqueen1990) [reply charts/rancher-aks-operator-crd/100.0.7+up1.0.8/templates/crds.yaml:127]: Makes sense! Thanks for clarifying!

**Review by stormqueen1990 — APPROVED** [review]

**Review by sbstp — APPROVED** [review]

**Review by jiaqiluo — COMMENTED** [review]
LGTM.
The final RC should be identical to the release version, so seeing the additional CRD changes in the PR does not look right. 
Communicated with @a-blender offline, and she will get confirmation from the highlander team for proof of validation.

**Comment by a-blender** [comment]
Based on @pgonin's feedback, this PR is being closed to tag aks operator v1.0.9 that will **not include** the aforementioned fixes because they are not required for rancher 2.6.x.

**Comment by a-blender** [comment]
Based on @pgonin's feedback, this PR is being closed to tag aks operator v1.0.9 that will not include the aforementioned fixes because they are not required for rancher 2.6.x.

**Pull request closed without merging**